src/main/java/com/zendesk/maxwell/schema/Database.java

```diff
@@ -48,6 +49,24 @@ public Table findTable(String name) {
 		return null;
 	}
 
+	public Table findTableOrThrow(String table) throws SchemaSyncError {
+		Table t = findTable(table);
+		if ( t == null )
+			throw new SchemaSyncError("Couldn't find table '" + table + "'" + " in database '" + this.name);
```

Comment:
This seems out of place... maybe make it part of SchemaChange? (it'd need to accept a database as param)